src/components/Comments/Comment.js

```diff
@@ -13,8 +13,9 @@ import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
 import MemoIc_downvote_on from '../../assets/arrow/Ic_downvote_on';
 import MemoIc_upvote_on from '../../assets/arrow/Ic_upvote_on';
 import useComment from './hooks/useComment';
-import {COLORS, FONTS} from '../../utils/theme';
+import {FONTS} from '../../utils/theme';
 import {calculateTime} from '../../utils/time';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated. Ensure that the new `colors` module provides all the necessary color constants that were previously available in `COLORS`. If not, this could lead to undefined values and potential errors or unexpected behavior.
```diff
- import {COLORS, FONTS} from '../../utils/theme';
+ import {FONTS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->